src/tm2/tm-conversions.h

```diff
@@ -0,0 +1,100 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2017 Intel Corporation. All Rights Reserved.
+#pragma once
+
+#include <map>
+#include "tm-device.h"
+#include "TrackingData.h"
+#include "stream.h"
+
+namespace librealsense
+{
+    const std::map<perc::PixelFormat, rs2_format> tm2_formats_map =
```

Comment:
perc namespace - a.k.a PerC ,  is it correct ? Can be renamed to librealsense convention

Reply:
cant change it for now, Once it will be fully integrated we will